nntrainer/tensor/blas_interface.cpp

```diff
@@ -55,15 +56,15 @@
   do {                                                                    \
     for (unsigned int m = 0; m < M; ++m) {                                \
       for (unsigned int n = 0; n < N; ++n) {                              \
-        _FP16 c = 0;                                                      \
+        float c = 0;                                                      \
         _FP16 c_old = C[m * ldc + n];                                     \
         for (unsigned int k = 0; k < K; ++k) {                            \
           _FP16 a, b;                                                     \
           a = ((TransA == CblasTrans) ? A[k * lda + m] : A[m * lda + k]); \
           b = ((TransB == CblasTrans) ? B[n * ldb + k] : B[k * ldb + n]); \
```

Comment:
```suggestion                                                 \
          _FP16 a = ((TransA == CblasTrans) ? A[k * lda + m] : A[m * lda + k]); \
          _FP16 b = ((TransB == CblasTrans) ? B[n * ldb + k] : B[k * ldb + n]); \
```
I know this isn't your work, though this might work better to possibly avoid Ahub issue.